pull_request_template.md

```diff
@@ -1,45 +1,21 @@
-<!--
-When lists are present, the item can be:
- - Deleted: The item is not applicable to the PR
- - Unchecked: The item is not done yet, but should be done as part of the PR
- - Checked: The item has been done
--->
-
-## Description
-
-<!-- 
-Please describe what your change is about. 
-If you made specific implementation choices worth an explanation, those can be detailed in this section 
--->
+### Description
 ...
 
-## Tests
-
-The following are included in this PR
-- [ ] Unit Tests (Most changes _should_ have unit tests)
-- [ ] Integration Tests
-
-## Checklist
-
-<!-- 
-This section contains a set of non-automated checks, it is there to remind you to think about some business critical topics. 
-If some are not applicable they could simply be deleted deleted.
-If you need to provide more details, please use the description section.
--->
-
-I have considered the following implications of my change: 
-
-- [ ] Security (in particular for changes to authentication, authorization, data fetching, ...)
-- [ ] Performance
-- [ ] Migration
-- [ ] Code maintainability
+### Checklist
+#### Source code
+- [ ] I've considered the implications on security (in particular for changes to authentication, authorization, data fetching, ...)
+- [ ] I've considered the implications on performances
+- [ ] I've considered the implications on migration
+- [ ] I've considered the implications on code maintainability
 
-## Documentation
+#### Tests
+- [ ] I've provided Unit Tests (Most changes _should_ have unit tests)
+- [ ] I've provided Integration Tests
```

Comment:
We should add something about documentation for manual tests associated with the change. Maybe something like "I've updated the parent issue to indicate manual validations needed for this change".

Reply:
Added a new bullet point for that